File: rest/taskrouter/workflows/instance/get/example-1/example-1.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
class Example 
{
  static void Main(string[] args) 
  {
    // Find your Account Sid and Auth Token at twilio.com/user/account
    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string AuthToken = "{{ auth_token }}";
    string WorkspaceSid = "{{ workspace_sid }}";
    string WorkflowSid = "{{ workflow_sid }}";
    var client = new TaskRouterClient(AccountSid, AuthToken);

    Workflow workflow = client.GetWorkflow(WorkspaceSid, WorkflowSid);
    Console.WriteLine(workflow.FriendlyName);
  }
}

File: rest/taskrouter/taskqueues/instance/post/example-1/example-1.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioTaskRouterClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token  = "{{ auth_token }}"
workspace_sid = "{{ workspace_sid }}"
taskqueue_sid = "{{ taskqueue_sid }}"

client = TwilioTaskRouterClient(account_sid, auth_token)

taskqueue = client.taskqueues(workspace_sid).update(taskqueue_sid, target_workers='languages HAS "english"')
print taskqueue.target_workers

# alternatively
taskqueue = client.workers(workspace_sid).get(taskqueue_sid)
taskqueue = taskqueue.update(target_workers='languages HAS "english"')
print taskqueue.target_workers